packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -3223,6 +3223,133 @@ describeWithDOM('mount', () => {
       expect(wrapper.isEmptyRender()).to.equal(data.expectResponse);
     });
 
+    describe('nested nodes', () => {
+      class RenderChildren extends React.Component {
+        render() {
+          return this.props.children;
+        }
+      }
+
+      class RenderNull extends React.Component {
+        render() {
+          return null;
+        }
+      }
+
+      describeIf(is('< 16'), 'rendered nodes', () => {
+        it('returns true for nested elements that return null', () => {
+          const wrapper = mount((
+            <RenderChildren>
+              <RenderNull />
+            </RenderChildren>
+          ));
+
+          expect(wrapper.isEmptyRender()).to.equal(true);
+        });
+
+        it('returns false for multiple nested elements that return all return null', () => {
+          const wrapper = mount((
+            <RenderChildren>
+              <div />
+            </RenderChildren>
+          ));
+
+          expect(wrapper.isEmptyRender()).to.equal(false);
+        });
+
+        it('returns false for multiple nested elements where one fringe returns a non null value', () => {
+          const wrapper = mount((
+            <RenderChildren>
+              <div>Hello</div>
+            </RenderChildren>
+          ));
+
+          expect(wrapper.isEmptyRender()).to.equal(false);
+        });
+
+        it('returns true for multiple nested elements where all values are null', () => {
+          const wrapper = mount((
+            <RenderChildren>
+              <RenderNull />
+            </RenderChildren>
+          ));
+
+          expect(wrapper.isEmptyRender()).to.equal(true);
+        });
+      });
+
+      describeIf(is('>= 16'), 'rendered nodes', () => {
+        it('returns true for nested elements that return null', () => {
```

Comment:
if the above describe runs unconditionally, this test becomes redundant and can be removed

Reply:
good catch.